app/javascript/app/document-capture/components/form-steps.jsx

```diff
@@ -38,27 +35,29 @@ function FormSteps({ steps, onComplete }) {
   }
 
   const { component: Component, name } = step;
+  const { isValid = () => true } = Component;
```

Comment:
Is it worth adding like throaway argument to this placeholder function? `isValid = (_values) => true`?

Reply:
> Is it worth adding like throaway argument to this placeholder function? `isValid = (_values) => true`?

I can see the potential benefit in making clearer how this function is intended to be implemented. But as proposed, it wouldn't pass lint:

![values is defined but never used](https://user-images.githubusercontent.com/1779930/88579587-44c86e00-d018-11ea-8902-509ee9dd0f77.png)

Another option could be to add type details.

For example:

```js
/** @type {{isValid:(values:object)=>boolean}} */
const { isValid = () => true } = Component;
```

...which has the added benefit of being something where we could eventually take advantage of type-checking:

![type checking error](https://user-images.githubusercontent.com/1779930/88579868-ba343e80-d018-11ea-943b-7bcf37f5c0a2.png)

Reply:
> Another option could be to add type details.

Added in 4550425.